compiler/rustc_feature/src/active.rs

```diff
@@ -650,6 +650,9 @@ declare_features! (
     /// Allows unsizing coercions in `const fn`.
     (active, const_fn_unsize, "1.53.0", Some(64992), None),
 
+    /// Allows `async {}` expressions in const contexts.
+    (active, const_async_blocks, "1.53.0", None, None),
```

Comment:
This should have a tracking issue

Reply:
Added https://github.com/rust-lang/rust/issues/85368